test/e2e/workload_api_test.go

```diff
@@ -100,7 +100,7 @@ func TestVSphereMulticlusterWorkloadClusterAPI(t *testing.T) {
 	test.DeleteManagementCluster()
 }
 
-func TestVSphereUpgradeLabelsTaintsUbuntuAPI(t *testing.T) {
```

Comment:
Are we not testing for Ubuntu at all then?

Reply:
Yes. We had decided to update some API tests to use BR rather than create a new test for each.